waiter/config-full.edn

```diff
@@ -690,14 +690,44 @@
                   ;; for sharing metrics between routers in the cluster:
                   :inter-router-metrics-idle-timeout-ms 2000
 
-                  ;; The interval (milliesconds) on which Waiter deletes
+                  ;; The interval (milliseconds) on which Waiter deletes
                   ;; metrics for services that are no longer running:
                   :metrics-gc-interval-ms 60000
 
                   ;; Waiter periodically triggers publishing of metrics with peer routers.
                   ;; This value (milliseconds) is used to throttle the rate of sending metrics:
                   :metrics-sync-interval-ms 50
 
+                  ;; Define optional reporters for codahale metrics.
+                  ;; Each reporter must have a :factory-fn key that specifies a factory method.
+                  ;; The factory method must create and start an instance of com.codahale.metrics.ScheduledReporter.
```

Comment:
Is this true? Also, is there a contract on the returned value of the factory function?

Reply:
that was old. changed to CodahaleReporter